Userland/DevTools/HackStudio/Editor.cpp

```diff
@@ -74,6 +72,22 @@ Editor::~Editor()
 {
 }
 
+void Editor::initialize_documentation_tooltip()
+{
+    m_documentation_tooltip_window = GUI::Window::construct();
+    m_documentation_tooltip_window->set_rect(0, 0, 500, 400);
+    m_documentation_tooltip_window->set_window_type(GUI::WindowType::Tooltip);
+    m_documentation_page_view = m_documentation_tooltip_window->set_main_widget<Web::OutOfProcessWebView>();
+}
+
+void Editor::initialize_parameters_hint_tooltip()
+{
+    m_parameters_hint_tooltip_window = GUI::Window::construct();
+    m_parameters_hint_tooltip_window->set_rect(0, 0, 280, 35);
```

Comment:
The size of the "parameters hint" tooltip window is currently hardcoded.
Is there currently a way to make it fit the actual html content?

Reply:
There is currently no such mechanism.